src/editor/editor_workspace/stage_workspace/stage_workspace_details_ui.go

```diff
@@ -104,6 +104,7 @@ func (dui *WorkspaceDetailsUI) setupFuncs() map[string]func(*document.Element) {
 		"searchEntityData":  dui.searchEntityData,
 		"addEntityData":     dui.addEntityData,
 		"changeData":        dui.changeData,
+		"removeEntityData":  dui.removeEntityData,
```

Comment:
This `removeEntityData` should be defined in the HTML (see my other comment for details).